ci-operator/step-registry/ipi/install/hosted-loki/ipi-install-hosted-loki-commands.sh

```diff
@@ -515,11 +515,7 @@ echo "Promtail manifests created, the cluster can be found at https://grafana-lo
 
 if [[ -f "/usr/bin/python3" ]]; then
   ENCODED_INVOKER="$(python3 -c "import urllib.parse; print(urllib.parse.quote('${OPENSHIFT_INSTALL_INVOKER}'))")"
-  cat >> ${ARTIFACT_DIR}/custom-link-loki.html << EOF
-  <html>
-  <body>
-  <a target="_blank" href="https://grafana-loki.ci.openshift.org/explore?orgId=1&left=%5B%22now-24h%22,%22now%22,%22Grafana%20Cloud%22,%7B%22expr%22:%22%7Binvoker%3D%5C%22${ENCODED_INVOKER}%5C%22%7D%22%7D%5D">Loki</a>
-  </body>
-  </html>
+  cat >> ${ARTIFACT_DIR}/custom-links.txt << EOF
+  <a target="_blank" href="https://grafana-loki.ci.openshift.org/explore?orgId=1&left=%5B%22now-24h%22,%22now%22,%22Grafana%20Cloud%22,%7B%22expr%22:%22%7Binvoker%3D%5C%22${ENCODED_INVOKER}%5C%22%7D%22%7D%5D">Loki<i class="material-icons" style="font-size: 1em; vertical-align: middle; padding-left: 3px;">open_in_new</i></a>
```

Comment:
Does this need an `| unpack` like the `echo` line from above?

Reply:
Good idea